configure.ac

```diff
@@ -229,6 +229,26 @@ if test "$with_valgrind" = yes ; then
     PKG_CHECK_MODULES(valgrind, valgrind)
 fi
 
+AC_ARG_WITH(valgrind-suppression,
+	      [Use an additional suppression file when running Valgrind],
+	      [with_valgrind_suppression=$withval],
+	      [with_valgrind_suppression=no])
+case "$with_valgrind_suppression" in
+yes)
+    SUPPRESSION_FILE="../../../np/valgrind.supp"
```

Comment:
Thanks for this patch.

NP is designed to be used without needing any Valgrind suppressions of it's own, and a lot of work went into fixing or working around Valgrind-discovered bugs in NP and in various libraries, and in some cases feeding back changes to the Valgrind team.  In particular I believe the `valgrind.supp` file in NP is unnecessary today.  Whether this approach is sustainable remains to be seen (it's already struggling in the Darwin port), and I would accept a patch to provide a default NP-specific suppression file if it seemed necessary.

However a default value of "../../../np/valgrind.np" for the pathname is not going to help anyone. Unless that pathname is available at runtime to wherever NP is run, it's just going to fail and confuse users.  So please either

- make the `--with-val-grind-suppression` option take a mandatory parameter, or
- install the default file into a location predictable at runtime for example `$(datarootdir)/$(PACKAGE_NAME)/valgrind.supp`